src/source/publishing/influx_publisher.cr

```diff
@@ -71,6 +71,8 @@ module PlaceOS::Source
       tags = HIERARCHY.each_with_object({} of String => String) do |key, obj|
         obj["pos_#{key}"] = data.zone_mapping[key]? || "_"
       end
+      tags["pos_system"] = data.control_system_id
+      tags["pos_index"] = data.index.to_i64.to_s
```

Comment:
I don't think we transform existing points if there is a change in the index of a module.
That might a thing to consider when setting up systems with analytics.